packages/backgammon.rb

```diff
@@ -0,0 +1,14 @@
+require 'package'
+require 'packages/libbsdgames2'
+
+class Backgammon < Bsdgame
+  description 'A text-based backgammon game; play against the computer or a friend.'
+  homepage $_bsdgame_homepage
+  version $_bsdgame_version
```

Comment:
We can set them in `libbsdgames2`

Reply:
If you know how, please tell me.

Reply:
`homepage 'https://www.polyomino.org.uk/computer/software/bsd-games/'`?
(since ruby doesn’t care the variable is set by which file)

Reply:
It's not a variable, it's a property function defined in `lib/package_helpers.rb` as far as I understand it.

Reply:
It didn't work @supechicken